neofetch

```diff
@@ -783,7 +783,7 @@ image_source="auto"
 # NOTE: AIX, Hash, Alpine, AlterLinux, Amazon, Anarchy, Android, instantOS,
 #       Antergos, antiX, "AOSC OS", "AOSC OS/Retro", Apricity, ArchCraft,
 #       ArcoLinux, ArchBox, ARCHlabs, ArchStrike, XFerience, ArchMerge, Arch,
-#       Artix, Arya, Bedrock, Bitrig, BlackArch, BLAG, BlankOn, BlueLight,
+#       Artix, Arya, Bedrock, Bitrig, BlackArch, BLAG, BlankOn, BlueLight, CalinixOS
```

Comment:
Please put CalinixOS in alphabetical order after Calculate and before Carbs

Reply:
Also since you added a _small variant, you can add that in the note 
![image](https://user-images.githubusercontent.com/22280294/182041533-0f446088-923c-42b6-b26e-a22eba781f53.png)

Reply:
Otherwise it's good!